static/src/javascripts/projects/common/modules/navigation/membership.js

```diff
@@ -0,0 +1,31 @@
+define(['common/modules/commercial/user-features', 'common/utils/fastdom-promise', 'Promise', 'common/utils/$'], function (userFeatures, fastdom, Promise, $) {
+    var LAST_CLASS = 'brand-bar__item--split--last';
+
+    function init() {
+        var refreshUser = new Promise(function (resolve) {
+            userFeatures.refresh();
+            resolve(userFeatures.isPayingMember());
```

Comment:
it doesn't look like either of these `userFeatures` functions involve I/O; is there any need for them to be in the promise chain?